realtime/pubSubStats.js

```diff
@@ -69,7 +70,7 @@ function track(src, evt, obj) {
     nameFromObj = obj.new.name;
     elapsed = now - new Date(obj.new.updatedAt);
   } else {
-    console.trace('Where is updatedAt? ' + JSON.stringify(obj));
+    logger.verbose('Where is updatedAt? ' + JSON.stringify(obj));
```

Comment:
add the stack trace e.g.
`logger.verbose('Where is updatedAt? ' + JSON.stringify(obj) + '\n' + new Error().stack);`